hyrax.gemspec

```diff
@@ -67,7 +67,7 @@ EOF
   spec.add_dependency 'active_attr', '~> 0.9.0'
   spec.add_dependency 'redlock', '>= 0.1.2'
   spec.add_dependency 'retriable', '>= 2.9', '< 4.0'
-  spec.add_dependency 'active-fedora', '>= 11.1.3'
+  spec.add_dependency 'active-fedora', '~> 11.2'
```

Comment:
I chose this one to pin to any `11.x` (using the `~>` operator).  I'm not attached any form in particular, I just find that one easier to read as an `11.x` constraint.